monodbassignment/posts.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id":"String datatype", 
+  "author_id": "String datatype(id of author)",
+   "timestamp":"timestamp datatype",
```

Comment:
Timestamp of what? You should be specific about what kind of timestamp it is. 
eg: created_at, posted_at, etc.